scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -86,7 +86,7 @@ class ExperimentalScanner(
         }
 
         val packageResults = if (packageScannerWrappers.isNotEmpty()) {
-            val packages = ortResult.getPackages().mapTo(mutableSetOf()) { it.pkg }
+            val packages = ortResult.getPackages().map { it.pkg }.filterConcluded().toSet()
```

Comment:
In Kotlin semantics, `filter { true }` defines what to keep (the "filtrate"), not what to "filter out". With that in mind, `filterConcluded()` sounds as if you'd keep packages with a concluded license, whereas the opposite is the case. So maybe call this `filterNotConcluded()` to resemble `filterNot`?